examples/simple_example.rs

```diff
@@ -0,0 +1,68 @@
+//! Centralized Telescope example with BLS signatures
+
+mod simple_threshold_signature;
+
+use crate::simple_threshold_signature::signature::Signature;
+use crate::simple_threshold_signature::signer::Signer;
+use crate::simple_threshold_signature::threshold_signature::ThresholdSignature;
+use alba::centralized_telescope::Telescope;
+use blst::min_sig::PublicKey;
+use rand_chacha::ChaCha20Rng;
+use rand_core::{RngCore, SeedableRng};
+
+const DATA_LENGTH: usize = 48;
+type Element = [u8; DATA_LENGTH];
+
+fn main() {
+    let mut rng = ChaCha20Rng::from_seed(Default::default());
+    let mut msg = [0u8; 16];
+    rng.fill_bytes(&mut msg);
+
+    println!("\n-------------- ALBA with Multi-Signature ---------------");
+    println!("--------------------------------------------------------");
+
+    // Define telescope parameters
+    let nb_elements: u64 = 1_000;
+    let soundness_param = 128.0;
+    let completeness_param = 128.0;
+    let set_size = nb_elements.saturating_mul(80).div_ceil(100);
+    let lower_bound = nb_elements.saturating_mul(20).div_ceil(100);
+
+    println!(" - Soundness parameter: {soundness_param}");
+    println!(" - Completeness parameter: {completeness_param}");
+    println!(" - Prover set size: {set_size}");
+    println!(" - Lower bound: {lower_bound}");
+
+    // Create the telescope structure
+    let alba = Telescope::create(soundness_param, completeness_param, set_size, lower_bound);
+
+    let mut public_key_list: Vec<(usize, PublicKey)> = Vec::with_capacity(nb_elements as usize);
```

Comment:
The `usize` variable always contains its own index, so it is redundant and can be removed.